test/sql/index/art/vacuum/test_art_vacuum_integers.test_slow

```diff
@@ -81,6 +81,12 @@ FROM pragma_database_size() AS current;
 statement ok
 DELETE FROM t WHERE i > 500000;
 
+# we disable verification for these last queries as verification stores multiple query results in main memory
+# query results are buffer-managed, so they add to the memory usage.
```

Comment:
Can we switch the verification code to force using the (old) in-memory result instead so it doesn't count against the memory limit?